dev/breeze/src/airflow_breeze/global_constants.py

```diff
@@ -102,18 +102,6 @@
     "MSSQL_VERSION": "MSSql version",
 }
 
-PARAM_NAME_FLAG = {
```

Comment:
Any reason this was removed?

Reply:
Yep. I found it was not used any more

Reply:
But yeah. Technically we should make it a separate PR (also the mysql versions fix. Let me split it.